src/components/scenes/newAccount/NewAccountTosScene.tsx

```diff
@@ -91,46 +111,20 @@ export const NewAccountTosScene = (props: Props) => {
         }
       }
     })
-    setTimeout(async () => {
-      try {
-        const abcAccount = await context.createAccount({
-          ...imports.accountOptions,
-          username,
-          password,
-          pin
-        })
-        abcAccount.watch('loggedIn', loggedIn => {
-          if (!loggedIn) dispatch({ type: 'RESET_APP' })
-        })
-        const touchDisabled = await isTouchDisabled(abcAccount)
-        if (!touchDisabled) {
-          await enableTouchId(abcAccount).catch(e => {
-            console.log(e) // Fail quietly
-          })
-        }
-        dispatch({ type: 'CREATE_ACCOUNT_SUCCESS', data: abcAccount })
-        dispatch({
-          type: 'NAVIGATE',
-          data: {
-            name: 'newAccountReview',
-            params: { ...route.params, account: abcAccount }
-          }
-        })
-        logEvent('Signup_Create_User_Success')
-        await abcAccount.dataStore.setItem(
-          Constants.OTP_REMINDER_STORE_NAME,
-          Constants.OTP_REMINDER_KEY_NAME_CREATED_AT,
-          Date.now().toString()
+    setTimeout(() => {
+      handleCreateAccount().catch((e: any) => {
+        console.error(e)
+        Alert.alert(
+          s.strings.create_account_error_title,
+          s.strings.create_account_error_message + '\n' + e.message,
+          [{ text: s.strings.ok }]
         )
-        dispatch(loadTouchState())
-      } catch (e: any) {
-        console.log(e)
-        dispatch({ type: 'CREATE_ACCOUNT_FAIL', data: e.message })
+        dispatch({ type: 'CLEAR_CREATE_ERROR_MESSAGE' })
```

Comment:
There is no need to dispatch `CLEAR_CREATE_ERROR_MESSAGE`, because there is no reducer to catch it. In fact, go ahead and delete the `CLEAR_CREATE_ERROR_MESSAGE` type definition as well.